code/datums/diseases/dna_spread.dm

```diff
@@ -38,7 +38,7 @@
 			if(prob(1))
 				affected_mob << "<span class='danger'>Your muscles ache.</span>"
 				if(prob(20))
-					affected_mob.take_organ_damage(1)
+					affected_mob.take_bodypart_damage(1)

```

Comment:
Fixing another proc using "organ" when it means bodyparts, less confusing that way.